src/store/actions/movie/movieActions.js

```diff
@@ -0,0 +1,110 @@
+import * as movieService from '../../../services/movieService.js';
+import * as movieActionsTypes from './movieActionTypes.js'
+import {commonActions} from '../../actions'
+
+export const loadMovies = () => async (dispatch, getState) => {
```

Comment:
I suggest to move action creators to different files base on usage area, e.g. list, delete etc

Reply:
move to common